src/main/java/lingogo/model/flashcard/UniqueFlashcardList.java

```diff
@@ -109,14 +110,15 @@ public void setFlashcards(List<Flashcard> flashcards) {
      * Exports the contents of internalList to a CSV file named {@code fileName}.
      * {@code fileName} must be a valid file name with .csv extension.
      */
-    public void downloadFlashcards(String fileName) throws CommandException {
+    public void exportFlashcards(String fileName) throws CommandException {
         String filePath = "./data/" + fileName;
         try {
             CSVWriter writer = new CSVWriter(new FileWriter(filePath));
-            String[] line = {"Foreign", "English"};
+            String[] line = {"Language", "Foreign", "English"};
```

Comment:
Maybe can bring out this string array as public static final String[], and call it `CSV_FORMAT` or something like that. See below also